app.js

```diff
@@ -205,9 +205,19 @@ io.sockets.on('connection', realtime.connection)
 
 // listen
 function startListen () {
-  server.listen(config.port, function () {
+  var host = config.host || '0.0.0.0'
```

Comment:
Can we move defaults to the config section where we usually handle defaults?

https://github.com/hackmdio/codimd/blob/master/lib/config/default.js